src/shared/telemetry/vscodeTelemetry.json

```
@@ -5,6 +5,17 @@
             "type": "string",
             "allowedValues": ["Create", "Update"],
             "description": "SSM Publish Document operation type"
+        },
+        {
+            "name": "documentFormat",
+            "type": "string",
+            "allowedValues": ["JSON, YAML", "Unselected"],
```

"None" is a more common name, use common names unless there is actually a reason to choose an uncommon name.

```suggestion
            "allowedValues": ["JSON, YAML", "None"],
```

Though I would suggest using something already defined in [here](https://github.com/aws/aws-toolkit-common/tree/master/telemetry/definitions) , but I didn't find anything.  @hunterwerlla  @bryceitoc9 do you recall if we have an existing pattern for "filetype" values in telemetry? IIRC normally it's just a string and we don't have hyper-specific types like this.

---

Since this is for the command execution instead of the actual file being interacted on, I think leaving as `undefined` as you suggested in comments below is the correct way to go. I'm assuming that there won't be any datapoints that are both `result: "Succeeded"` and `documentFormat: undefined`, right?